src/app/views/censusdata/create.html

```diff
@@ -0,0 +1,171 @@
+            <li>Always provide "number of adults and juveniles" data for species.</li>
+            <li>Always observe birds from at least 50 yards away.</li>
+        </ul>
+    </div>
+                
+    <form class="form-horizontal" role="form"> 
+        <div class="form-group panel panel-primary">
+            <div class="panel-heading">Section One</div>
+            <div class="panel-body">              
+            <div class="form-group">
+                <label class="control-label col-md-3" for="colonyCode">Colony Code:</label>
+                <div class="col-md-3">
+                <input type="text" class="form-control" id="colonyCode" placeholder="Enter Colony Code">
+                </div>    
+            </div> 
+            <div class="form-group">
+                <label class="control-label col-md-3" for="colonyName">Colony Name:</label>
+                <div class="col-md-9">
+                <input type="text" class="form-control" id="colonyName" placeholder="Enter Colony Name">
+                </div>
+            </div>
+            <div class="form-group">
+                <label class="control-label col-md-3" for="surveyDate">Survey Date:</label>
+                <div class="col-md-4">
+                <input type="date" class="form-control" id="surveyDate" placeholder="Enter Survey Date">
+                </div>
+            </div>   
+            <div class="form-group">
+                <label class="control-label col-md-3" for="timeStart">Time Start:</label>
+                <div class="col-md-3">
+                <input type="time" class="form-control" id="timeStart" placeholder="Enter Survey Date">
+                </div>
+                <label class="control-label col-md-3" for="timeEnd">Time End:</label>
+                <div class="col-md-3">
+                <input type="time" class="form-control" id="timeEnd" placeholder="Enter Survey Date">
+                </div>
+            </div>   
+            <div class="form-group">
+                <label class="control-label col-md-3" for="latitude">Latitude:</label>
+                <div class="col-md-3">
+                <input type="text" class="form-control" id="latitude" placeholder="Enter Latitude">
+                </div>    
+            </div> 
+            <div class="form-group">
+                <label class="control-label col-md-3" for="longitude">Longitude:</label>
+                <div class="col-md-3">
+                <input type="text" class="form-control" id="longitude" placeholder="Enter Longitude">
+                </div>    
+            </div> 
+            <div class="form-group">
+                <label class="control-label col-md-3">Site Type:</label>
+                <div class="col-md-9">
+                <div class="checkbox">
+                    <label ng-repeat="siteType in siteTypeInfo"><input type="checkbox" value="{{siteType.Id}}"> {{ siteType.Description }} &nbsp;</label>        
```

Comment:
Let's change these checkboxes to radio buttons for now, unless we get clarification otherwise.